zk/conn.go

```diff
@@ -964,6 +961,25 @@ func (c *Conn) queueRequest(opcode int32, req interface{}, res interface{}, recv
 	return rq.recvChan
 }
 
+func (c *Conn) queueRequestWithTimeout(opcode int32, req interface{}, res interface{}, recvFunc func(*request, *responseHeader, error), timeout time.Duration) <-chan response {
```

Comment:
since recvFunc is nil why have this as an argument? same comment for the request/response.

Reply:
Keeping this parameter so that the new function is compatible with queueRequest().